<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomTransferItem</class>
 <widget class="QWidget" name="CustomTransferItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">CustomTransferItem
{
	background-color: transparent;
}

#lTransferType, #lTransferTypeActive, #lFileType, #lFileTypeActive, #lGetLink, #lFileTypeCompleted, #lTransferTypeCompleted, #pbTransfer, #bTransferType
{
	border: none;
}

#lFileName, #lFileNameCompleted
{
  font-family: &quot;Source Sans Pro&quot;;
  font-size: 13px;
  color: #333333;
}

#lSpeed
{
  font-family: &quot;Source Sans Pro&quot;;
  font-size: 12px;
  color: #666666;
}

#lTransfers
{
  font-family: &quot;Source Sans Pro&quot;;
  font-size: 12px;
  color: #666666;
}

#lElapsedTime
{
  font-family: &quot;Source Sans Pro&quot;;
  font-size: 12px;
  color: #999999;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sTransferState">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>60</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="completedTransfer">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>9</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="lFileTypeCompleted">
         <property name="minimumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wTransferInfo" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lFileNameCompleted">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>220</width>
              <height>17</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>8</number>
            </property>
            <item>
             <widget class="QPushButton" name="lTransferTypeCompleted">
              <property name="minimumSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lElapsedTime">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="lGetLink">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="activeTransfer">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wContainer" native="true">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="wVisualData" native="true">
               <property name="minimumSize">
                <size>
                 <width>350</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>400</width>
                 <height>16777215</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>9</number>
                </property>
                <property name="leftMargin">
                 <number>6</number>
                </property>
                <property name="topMargin">
                 <number>2</number>
                </property>
                <property name="rightMargin">
                 <number>16</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="lFileType">
                  <property name="minimumSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>48</width>
                    <height>48</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="wTransferItem" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>8</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>5</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="lFileName">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>17</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>Source Sans Pro</family>
                      </font>
                     </property>
                     <property name="focusPolicy">
                      <enum>Qt::StrongFocus</enum>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QWidget" name="wSpeed" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>15</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>15</height>
                      </size>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_2">
                      <property name="spacing">
                       <number>5</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QPushButton" name="lTransferType">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>12</width>
                          <height>12</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>12</width>
                          <height>12</height>
                         </size>
                        </property>
                        <property name="focusPolicy">
                         <enum>Qt::NoFocus</enum>
                        </property>
                        <property name="styleSheet">
                         <string notr="true">border: none;</string>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>12</width>
                          <height>12</height>
                         </size>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="lTransfers">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>15</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>15</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QLabel" name="lSpeed">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>15</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>16777215</width>
                          <height>15</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="bClockDown">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../Resources_win.qrc">
                    <normaloff>:/images/clock_ico.png</normaloff>:/images/clock_ico.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="lRemainingTime">
                  <property name="minimumSize">
                   <size>
                    <width>64</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>64</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="wProgressBar" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QProgressBar" name="pbTransfer">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>350</width>
                    <height>2</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>400</width>
                    <height>2</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                  <property name="textVisible">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_win.qrc"/>
 </resources>
 <connections/>
</ui>
